test/integration/static_pages_test.rb

```diff
@@ -0,0 +1,22 @@
+require "test_helper"
+
+class StaticPagesTest < ActionDispatch::IntegrationTest
+  include Capybara::DSL
+
+  test "a guest can visit the about page in the footer on any page" do
+    user = create(:user)
```

Comment:
This does not test if a guest can visit the page and sees a footer. There is no test to see if footer exists if visitor is an admin or user.